Sources/PriorityQueue/PriorityQueue+Sequence.swift

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension PriorityQueue:Sequence{
+    
+    /*
+     *  Defines an iterator to conform to Sequence
+     */
+    public struct Iterator:IteratorProtocol{
+        //the priority queue at a specific moment
+        private var snapshotPriorityQueue:PriorityQueue
```

Comment:
This property does not need to be mutable.

The name `snapshotPriorityQueue` is needlessly verbose -- I recommend something simple like `_base` instead.

```suggestion
        private let _base: PriorityQueue
```

You do not need to emphasize that this property contains a priority queue at a specific moment. That's true for every variable.